catalog/camel-report-maven-plugin/src/main/java/org/apache/camel/maven/ValidateMojo.java

```diff
@@ -746,6 +691,68 @@ private int validateEndpointPairs(List<CamelEndpointDetails> endpoints, String s
         return errors;
     }
 
+    private StringBuilder buildValidationSuccessMessage(String str, String className, String lineNumber, String methodName, String fileName, String uri) {
+        StringBuilder sb = new StringBuilder();
+        sb.append(str);
+        if (className != null && lineNumber != null) {
+            // this is from java code
+            sb.append(className);
+            if (methodName != null) {
+                sb.append(".").append(methodName);
```

Comment:
Ideally if you could replace the regular expression `append\("(.)"\)` with `append('$1')` globally it could nice

Reply:
OK